### PR TITLE
change license from gnu to mit

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

**Comment by radius314** [comment]
looks like this supercedes #175

**Comment by pjaudiomv** [comment]
totally forgot about that pr, i can add additional commits for other link changes to this one.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ scheduled weekly, recurring events.
 The intended clientele is Narcotics Anonymous Service bodies. They implement a BMLT
```
Comment (radius314) [inline README.md]: clientele?

Comment (pjaudiomv) [reply README.md]: user?

Comment (pjaudiomv) [reply README.md]: The intended users our Narcotics Anonymous Service bodies.

**Pull request merged**